pkg/crd/kubermatic/v1/settings.go

```diff
@@ -102,6 +103,20 @@ type MlaOptions struct {
 	MonitoringEnforced bool `json:"monitoringEnforced"`
 }
 
+type ClusterTemplateOptions struct {
+	// Predefined cluster template sizes
+	Sizes []ClusterTemplateSize `json:"clusterTemplateSizes"`
+}
+
+type ClusterTemplateSize struct {
+	// Size name. Can be for example: small, large, L, XL
+	Name string `json:"name"`
```

Comment:
Are there any restrictions for the name? If it can be similar to the clusters then I'd avoid giving examples. It can be misleading.